rustls/src/crypto/cipher.rs

```diff
@@ -171,6 +171,12 @@ impl From<[u8; NONCE_LEN]> for Iv {
     }
 }
 
+impl AsRef<[u8]> for Iv {
```

Comment:
Can/should we remove the `pub(crate)` from `Iv.0` here?

Reply:
Done.